ui/components/app/security-provider-banner-message/security-provider-banner-message.test.js

```diff
@@ -0,0 +1,121 @@
+import { fireEvent } from '@testing-library/react';
+import React from 'react';
+import configureMockStore from 'redux-mock-store';
+import { renderWithProvider } from '../../../../test/lib/render-helpers';
+import SecurityProviderBannerMessage from './security-provider-banner-message';
+
+describe('Security Provider Banner Message', () => {
+  const store = configureMockStore()({});
+
+  const thisIsBasedOnText = 'This is based on information from';
+
+  it('should render SecurityProviderBannerMessage component properly when flagAsDangerous is 1', () => {
+    const securityProviderResponse = {
+      flagAsDangerous: 1,
```

Comment:
Can we use named constants here, 0, 1, 2 doesn't mean much and could be confusing maintaining this in the future.

Reply:
Fixed